cluster/images/etcd/rollback/README.md

```diff
@@ -1,45 +0,0 @@
-# Rollback workflow
-
-Build it in this directory.
```

Comment:
Are you planning to add another README file as documentation?

Reply:
We'll consolidate docs to the `cluster/images/etcd/README.md` file.  I'll do another pass on it shortly to make sure it's fleshed out.